### PR TITLE
Do-over: chapter for Studio work

### PR DESCRIPTION
**Pull request by lamagnifica**
I made a mess of my branches, so starting over. 
@lou-wang, @tymofij, @polesye, @olmar, @remyedX, and @uhanile this chapter has been reworked significantly to reduce content to only what is different about the process for course teams using the new workflow. Reviewed thoroughly by @rjgrubb.
@nasthagiri, @frrrances, @WatsonEmily , @srpearce , @mhoeber, @catong FYI.

### CODE REVIEW

**Comment by lamagnifica** [comment]
@rjgrubb , @lou-wang , with your thumbs on this PR, the initial version of the "Video Processing" guide will be ready for limited release (for example, to @remyedX and the PMs).

**Comment by lamagnifica** [comment]
Apologies, I don't know what is going on with this PR. The only file that needs to be reviewed at this point is the video_uploads.rst file. Thanks for bearing with me on this.
http://draft-processing-video-files.readthedocs.org/en/latest/video_uploads.html

**Comment by lou-wang** [comment]
Generally looks good. I would take out "Output: Transfer to edX Studio" because that's not a property of the incoming video file. For codec, I would just leave "H.264" as .mp4 is technically a container format, not a codec. "Aspect: 1.0" is weird to me - isn't this supposed to be 16:9? @rjgrubb would know best here.

**Comment by rjgrubb** [comment]
That’s fine to remove the first row. Know what you mean about the codec,
but we kept the .mp4 to encourage .mp4 files. But I agree, the codec is
H.264. MP4 is just the file container. Let’s change to: Codec & Container:
H.264, mp4

The resolution or frame size (1920x1080) is the 16x9. Aspect refers to the
pixel aspect ratio. Square Pixels (1.0) is for a standard-definition or
high-definition file created using square pixels. Important to have square
px for video, but let’s change "Aspect" to “Pixel Aspect Ratio” to remove
any questions.

On Fri, Dec 12, 2014 at 1:56 PM, lou-wang notifications@github.com wrote:

> Generally looks good. I would take out "Output: Transfer to edX Studio"
> because that's not a property of the incoming video file. For codec, I
> would just leave "H.264" as .mp4 is technically a container format, not a
> codec. "Aspect: 1.0" is weird to me - isn't this supposed to be 16:9?
> @rjgrubb https://github.com/rjgrubb would know best here.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/edx/edx-documentation/pull/58#issuecomment-66817587.

## 

Rachel Grubb
edX | Media Coordinator
141 Portland Street, 8th Floor
Cambridge, MA 02139
rachel@edx.org

File: en_us/video/source/video_uploads.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+     - 1.0
+   * - Bit Rate
+     - VBR, 2 pass
+   * - Target VBR
+     - 5 mbps
+   * - Max VBR
+     - 7.5 mbps
+   * - Audio
+     - AAC 44.1 / 192 kbps
+
```
Comment (rjgrubb) [inline en_us/video/source/video_uploads.rst:62]: In table, remove first row: output
Revise second row to: Codec & Container: H.264, mp4 
In 4th row, change “Aspect” to “Pixel Aspect Ratio”

**Pull request closed without merging**